// @bgd-labs/aave-address-book/dist/ChainlinkMantle.mjs
// src/ts/ChainlinkMantle.ts
var FBTC_BTC_Exchange_Rate = "0x2D787621f9C635ADe313126dF7615c02bff35517";
var USDe_USD = "0x5166FC3adff16E99bb099834a1315e57C5444394";
var OUSDT_USD = "0xC5fD97adABf827e9FCC4ed936360f47080F35868";
var L2_Sequencer_Uptime_Status_Feed = "0xaDE1b9AbB98c6A542E4B49db2588a3Ec4bF7Cdf0";
var ETH_USD = "0x5bc7Cf88EB131DB18b5d7930e793095140799aD5";
var USDC_USD = "0x22b422CECb0D4Bd5afF3EA999b048FA17F5263bD";
var LINK_USD = "0x5871AdBEEdAD531C68A8FD32fE86f07d6b4C645d";
var ynETHx_ETH_Exchange_Rate = "0xE2f6c76c05985e90B043CC6ddf1D09e8BF5FAAf9";
var Calculated_mETH_USD = "0xB16FcAFB8378baA0a69142a325878FDCad58606A";
var mETH_ETH = "0x6352E3A925E27130317d3A0A5b0b0303e9f2fc7c";
var Calculated_FBTC_USD = "0x73b15e19b247263D03D7938f1356304b7B330Ff0";
var sUSDe_USD = "0x6d5110FB8F6a65c46B89a64C9ac7E3542D31AbA3";
var BTC_USD = "0x7db2275279F52D0914A481e14c4Ce5a59705A25b";
var USD1_USD = "0x01dd982820B7c25e2A9C77BDd3bC0eDA90298A2F";
var MNT_USD = "0xD97F20bEbeD74e8144134C4b148fE93417dd0F96";
var USDT_USD = "0xd86048D5e4fe96157CE03Ae519A9045bEDaa6551";
var CHAIN_ID = 5e3;
export {
  BTC_USD,
  CHAIN_ID,
  Calculated_FBTC_USD,
  Calculated_mETH_USD,
  ETH_USD,
  FBTC_BTC_Exchange_Rate,
  L2_Sequencer_Uptime_Status_Feed,
  LINK_USD,
  MNT_USD,
  OUSDT_USD,
  USD1_USD,
  USDC_USD,
  USDT_USD,
  USDe_USD,
  mETH_ETH,
  sUSDe_USD,
  ynETHx_ETH_Exchange_Rate
};
