// @bgd-labs/aave-address-book/dist/ChainlinkLinea.mjs
// src/ts/ChainlinkLinea.ts
var ETH_USD = "0x3c6Cd9Cc7c7a4c2Cf5a82734CD249D7D593354dA";
var LINK_USD = "0x8dF01C2eFed1404872b54a69f40a57FeC1545998";
var WSTETH_USD = "0x8eCE1AbA32716FdDe8D6482bfd88E9a0ee01f565";
var FOXY_USD = "0xdE14081b6bd39230EcA7Be1137413b7b87B07C07";
var weETH_ETH = "0xC4bF21Ab46bd22Cf993c0AAa363577bD2Af83544";
var agETH_ETH_Exchange_Rate = "0x5C5Ee01b351b7ef0b16Cfd59E93F743E0679d7bC";
var osETH_ETH_Exchange_Rate = "0x4C2f2CD982DD7fF2a607dcEB0B30fae8208D7156";
var inETH_ETH_Exchange_Rate = "0x1EBA8E11f5Dcc43ae9A8cB7A8346D1d191671BAe";
var AAVE_USD = "0x09B0a8AFD9185500d7C64FC68338b4C50db6df1d";
var ultraETHs_ETH_Exchange_Rate = "0x0956106Bd02f2C7eca0AbDe1d51Fc8050CC41714";
var COMP_USD = "0xc0068A2F7e4847DF9C3A34B27CCc07b7e15e0458";
var wrsETH_rsETH_Exchange_Rate = "0xEEDF0B095B5dfe75F3881Cb26c19DA209A27463a";
var ARB_USD = "0x28606F10277Cc2e99e57ae2C55D26860E13A1BBD";
var instETH_ETH_Exchange_Rate = "0xeeD6246A489f386b10eeB2908239C8529B9b25DD";
var LINK_ETH = "0xc4194f19E3a0836F6B998394445C6535c50604Ce";
var USDC_USD = "0xAADAa473C1bDF7317ec07c915680Af29DeBfdCb5";
var BTC_USD = "0x7A99092816C8BD5ec8ba229e3a6E6Da1E628E1F9";
var EUR_USD = "0x637cf12017219Dd3A758818eD63185f7acF7D935";
var ezETH_ETH = "0xB1d9A4Fe9331E28C5588B63343BF064A397aadB8";
var DAI_USD = "0x5133D67c38AFbdd02997c14Abd8d83676B4e309A";
var POL_USD = "0xEF77B4A7D92eBDC89025B8E11916A69BDA6d189c";
var ynETH_ETH_Exchange_Rate = "0x57011f81c93Da4dFb244c80667aAECf8cA9D53Bb";
var USDT_USD = "0xefCA2bbe0EdD0E22b2e0d2F8248E99F4bEf4A7dB";
var weETH_eETH_Exchange_Rate = "0x1FBc7d24654b10c71fd74d3730d9Df17836181EF";
var ezETH_ETH_Exchange_Rate = "0xb71F79770BA599940F454c70e63d4DE0E8606731";
var rswETH_ETH_Exchange_Rate = "0xB21b5Abc612b8045A33fC421d611D55473E012BD";
var wstETH_stETH_Exchange_Rate = "0x3C8A95F2264bB3b52156c766b738357008d87cB7";
var RSETH_ETH = "0x85342bC62aadef58f029ab6d17D643949e6F073e";
var MATIC_USD = "0x9ce4473B42a639d010eD741df3CA829E6e480803";
var CHAIN_ID = 59144;
export {
  AAVE_USD,
  ARB_USD,
  BTC_USD,
  CHAIN_ID,
  COMP_USD,
  DAI_USD,
  ETH_USD,
  EUR_USD,
  FOXY_USD,
  LINK_ETH,
  LINK_USD,
  MATIC_USD,
  POL_USD,
  RSETH_ETH,
  USDC_USD,
  USDT_USD,
  WSTETH_USD,
  agETH_ETH_Exchange_Rate,
  ezETH_ETH,
  ezETH_ETH_Exchange_Rate,
  inETH_ETH_Exchange_Rate,
  instETH_ETH_Exchange_Rate,
  osETH_ETH_Exchange_Rate,
  rswETH_ETH_Exchange_Rate,
  ultraETHs_ETH_Exchange_Rate,
  weETH_ETH,
  weETH_eETH_Exchange_Rate,
  wrsETH_rsETH_Exchange_Rate,
  wstETH_stETH_Exchange_Rate,
  ynETH_ETH_Exchange_Rate
};
