// @bgd-labs/aave-address-book/dist/ChainlinkZkSync.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/ts/ChainlinkZkSync.ts
var ChainlinkZkSync_exports = {};
__export(ChainlinkZkSync_exports, {
  AAVE_USD: () => AAVE_USD,
  BTC_USD: () => BTC_USD,
  CHAIN_ID: () => CHAIN_ID,
  DAI_USD: () => DAI_USD,
  DOGE_USD: () => DOGE_USD,
  ETH_USD: () => ETH_USD,
  Fidelity_USD_NAV: () => Fidelity_USD_NAV,
  L2_Sequencer_Uptime_Status_Feed: () => L2_Sequencer_Uptime_Status_Feed,
  LINK_ETH: () => LINK_ETH,
  LINK_USD: () => LINK_USD,
  MELANIA_USD: () => MELANIA_USD,
  PEPE_USD: () => PEPE_USD,
  SOL_USD: () => SOL_USD,
  TRUMP_USD: () => TRUMP_USD,
  UNI_USD: () => UNI_USD,
  USDC_USD: () => USDC_USD,
  USDM_USD: () => USDM_USD,
  USDT_USD: () => USDT_USD,
  USDe_USD: () => USDe_USD,
  ZK_USD: () => ZK_USD,
  ezETH_ETH: () => ezETH_ETH,
  rsETH_ETH_Exchange_Rate: () => rsETH_ETH_Exchange_Rate,
  rswETH_ETH_Exchange_Rate: () => rswETH_ETH_Exchange_Rate,
  sUSDe_USDe_Exchange_Rate: () => sUSDe_USDe_Exchange_Rate,
  weETH_eETH_Exchange_Rate: () => weETH_eETH_Exchange_Rate,
  wrsETH_ETH_Exchange_Rate_ETH: () => wrsETH_ETH_Exchange_Rate_ETH,
  wstETH_stETH_Exchange_Rate: () => wstETH_stETH_Exchange_Rate
});
module.exports = __toCommonJS(ChainlinkZkSync_exports);
var ezETH_ETH = "0x600E54Fc4cF6148Cd6696A3D37F5f672829BF7C2";
var PEPE_USD = "0x7a6333CaC589e9B11b1fEC190a5828272A2893B5";
var L2_Sequencer_Uptime_Status_Feed = "0x0E6AC8B967393dcD3D36677c126976157F993940";
var LINK_USD = "0x1b5a683579f53b9E30B538F70544444389633c75";
var wrsETH_ETH_Exchange_Rate_ETH = "0x78F211E6707463F5b2254DE667b9A70a95d816b5";
var LINK_ETH = "0xB66325FC0F8aA6dE6FeDFF4e51e54025cEea51eE";
var AAVE_USD = "0x2137c69DCb41f611Cc8f39F8A98047e774d6ED74";
var weETH_eETH_Exchange_Rate = "0x8D3184a992f93729b249407C33F1e78abE0d650e";
var sUSDe_USDe_Exchange_Rate = "0x97920183c36B022B46D6C14b9dA36c5f31A98C6A";
var USDC_USD = "0x1824D297C6d6D311A204495277B63e943C2D376E";
var UNI_USD = "0x93A08A9D592101938D4a63c86d0989d7018c00D9";
var wstETH_stETH_Exchange_Rate = "0x24a0C9404101A8d7497676BE12F10aEa356bAC28";
var rswETH_ETH_Exchange_Rate = "0x92c2ed09B096470DF40973231Dac5aC987F9b8F9";
var MELANIA_USD = "0x20Af81052722DbE38792e595De231C41F4D3255B";
var DAI_USD = "0x5d336664b5D7A332Cd256Bf68CbF2270C6202fc6";
var DOGE_USD = "0x2cC24D99500a134ea7f78736b5C329C84599fb1B";
var ETH_USD = "0x6D41d1dc818112880b40e26BD6FD347E41008eDA";
var USDe_USD = "0x4899faF0b6c36620168D00e3DbD4CB9361244c4d";
var Fidelity_USD_NAV = "0xC406104c42211abd1A2cD411DDd071511515bDdd";
var BTC_USD = "0x4Cba285c15e3B540C474A114a7b135193e4f1EA6";
var ZK_USD = "0xD1ce60dc8AE060DDD17cA8716C96f193bC88DD13";
var TRUMP_USD = "0x960eA047617cfB29AFcf2Ef6794C76713ff0C20C";
var USDM_USD = "0x6Ab6c24f9312a6cB458761143D373A8f11573C4B";
var USDT_USD = "0xB615075979AE1836B476F651f1eB79f0Cd3956a9";
var rsETH_ETH_Exchange_Rate = "0x7024c64Ad30Ebf224e417CfDE4438606d2b9B690";
var SOL_USD = "0x498232F0a52D4e94A6e0Ea93D63C07Bc63133009";
var CHAIN_ID = 324;
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  AAVE_USD,
  BTC_USD,
  CHAIN_ID,
  DAI_USD,
  DOGE_USD,
  ETH_USD,
  Fidelity_USD_NAV,
  L2_Sequencer_Uptime_Status_Feed,
  LINK_ETH,
  LINK_USD,
  MELANIA_USD,
  PEPE_USD,
  SOL_USD,
  TRUMP_USD,
  UNI_USD,
  USDC_USD,
  USDM_USD,
  USDT_USD,
  USDe_USD,
  ZK_USD,
  ezETH_ETH,
  rsETH_ETH_Exchange_Rate,
  rswETH_ETH_Exchange_Rate,
  sUSDe_USDe_Exchange_Rate,
  weETH_eETH_Exchange_Rate,
  wrsETH_ETH_Exchange_Rate_ETH,
  wstETH_stETH_Exchange_Rate
});
